packages/oidc-client/src/initWorker.ts

```diff
@@ -72,23 +84,13 @@ const sendMessageAsync =
         messageChannel.port1.close();
         messageChannel.port2.close();
       };
-      registration.active.postMessage(data, [messageChannel.port2]);
+      registration.active.postMessage({ ...data, tabId: getTabId(data.configurationName) }, [
```

Comment:
this was very annoying that some actions has `tabId` and some are missing. I decided to unify this piece, so any interaction between client(s)/serviceWorker will hold ID for identification which tab performs specific action

Reply:
Thank you @wermanoid , very sorry for the delay. I am in holidays :)

Reply:
I can publish it as an alpha if you want ?

Reply:
sure, let's do it :)

thanks!

Reply:
Published as an alpha @wermanoid thank you !